p4-16/psa/PSA.mdk

```diff
@@ -299,6 +313,71 @@ copy going to the control CPU to have a software-defined header
 (defined in the P4 program) that is different than any headers on the
 packet copies going to the Egress control block.
 
+A PSA implementation may implement multiple classes of service for
+packets sent to the packet buffer.  If so, the Ingress control block
+may choose to assign a value to the `ostd.class_of_service` field to
+change the packet's class of service to a value other than the default
+of 0.
+
+PSA only specifies how the Ingress control block can choose the class
+of service the packet is assigned.  PSA does not mandate a scheduling
```

Comment:
It seems to me that with the restriction to just Ingress, defining a BQE block at the end of the pipeline is not very useful. We still define drop/truncate on it such that we have symmetrical functionality for these operations with the Ingress, so we may want to keep it. But then, maybe we should rename it to something that matches the reduced functionality.